test/tools/ossfuzz/protoToAbiV2.cpp

```diff
@@ -1,747 +1,239 @@
-#include <regex>
-#include <numeric>
-#include <boost/range/adaptor/reversed.hpp>
 #include <test/tools/ossfuzz/protoToAbiV2.h>
-#include <libdevcore/StringUtils.h>
-#include <libdevcore/Whiskers.h>
-#include <liblangutil/Exceptions.h>
 
 using namespace std;
 using namespace dev;
 using namespace dev::test::abiv2fuzzer;
 
-// Create a new variable declaration and append said variable to function parameter lists
-// of coder functions.
-// Declared name is x_<i>; parameterized name is c_<i>
-// where <i> is a monotonically increasing integer.
-void ProtoConverter::createDeclAndParamList(
-	std::string const& _type,
-	DataType _dataType,
-	std::string& _varName,
-	std::string& _paramName
-)
-{
-	auto varNames = newVarNames(getNextVarCounter());
-	_varName = varNames.first;
-	_paramName = varNames.second;
-
-	// Declare array
-	appendVarDeclToOutput(_type, _varName, getQualifier(_dataType));
-
-	// Add typed params for calling public and external functions with said type
-	appendTypedParams(
-		CalleeType::PUBLIC,
-		isValueType(_dataType),
-		_type,
-		_paramName,
-		((m_varCounter == 1) ? Delimiter::SKIP : Delimiter::ADD)
-	);
-	appendTypedParams(
-		CalleeType::EXTERNAL,
-		isValueType(_dataType),
-		_type,
-		_paramName,
-		((m_varCounter == 1) ? Delimiter::SKIP : Delimiter::ADD)
-	);
-}
-
-void ProtoConverter::visitArrayType(std::string const& _baseType, ArrayType const& _x)
-{
-	std::string type = arrayTypeAsString(_baseType, _x);
-	std::string varName, paramName;
-	createDeclAndParamList(type, DataType::ARRAY, varName, paramName);
-	// Resize-initialize array and add checks
-	resizeInitArray(_x, _baseType, varName, paramName);
-}
-
-void ProtoConverter::visitType(
-	DataType _dataType,
-	std::string const& _type,
-	std::string const& _value
-)
-{
-	std::string varName, paramName;
-	createDeclAndParamList(_type, _dataType, varName, paramName);
-	addCheckedVarDef(_dataType, varName, paramName, _value);
-}
-
-void ProtoConverter::appendVarDeclToOutput(
-	std::string const& _type,
-	std::string const& _varName,
-	std::string const& _qualifier
+string ProtoConverter::appendVarDeclToOutput(
```

Comment:
This is now rather ``getVarDecl``, isn't it?

Reply:
Thanks, will rename it.